test/verify/machineslib.py

```diff
@@ -824,6 +824,10 @@ def testVCPU(self):
         b.click("#vm-subVmTest1-run")
         b.wait_in_text("#vm-subVmTest1-state", "running")
 
+        # Wait for the VCPUs link to get new values before opening the dialog
+        b.wait_visible("#vm-subVmTest1-vcpus-count")
```

Comment:
Heh, this is not gonna be needed when #11688 lands :)

Reply:
I will wait for that to land first then.

Reply:
I'd rather land these flake fixes first, they (also) block #11688, and I think the latter might need more work. The extra wait_visible won't hurt.